draft-parecki-oauth-client-id-metadata-document.md

```diff
@@ -69,28 +72,27 @@ at each authorization server.
 
 {::boilerplate bcp14-tagged}
 
-
 # Client Identifier
 
-This specification defines the client identifier as a URL with
-the following restrictions. Client identifier URLs MUST have
-an "https" scheme, MUST contain a path component, MUST NOT
-contain single-dot or double-dot path segments, MAY contain a query
-string component, MUST NOT contain a fragment component, MUST NOT
-contain a username or password component, and MAY contain a port.
+This specification defines the client identifier as a URL with the following
+restrictions. Client identifier URLs MUST have an "https" scheme, MUST contain a
+path component, MUST NOT contain single-dot or double-dot path segments, MUST
+NOT contain a fragment component and MUST NOT contain a username or password
+component. Client identifier URLs MAY contain a query string component and MAY
+contain a port.
 
 This specification places no restrictions on what URL is used as
 a client identifier. A short URL is RECOMMENDED, since the URL may
 be displayed to the end user in the authorization interface or in
-management interfaces.
-
+management interfaces. Usage of a stable URL that does not frequently
+change for the client is also RECOMMENDED.
```

Comment:
Would like to find some better language here, maybe referencing [Cool URIs](https://www.w3.org/Provider/Style/URI) is worthwhile?